src/MainWindow.h

```diff
@@ -158,9 +158,10 @@ private slots:
 
 private slots:
 	void selectFindType(int);
-	void find();
-	void findString(QString);
-	void findAndReplace();
+    void hideFind();
```

Comment:
Would you mind using the same indentation style as the existing code (single tab indent)? I'm aware that this isn't the optimal solution, but consistency is important.
This applies to all changes.